azure-ipam/linux.Dockerfile

```diff
@@ -0,0 +1,32 @@
+ARG ARCH
+ARG DROPGZ_VERSION=v0.0.12
+ARG OS_VERSION
```

Comment:
Are we using the OS_VERSION in this dockerfile ?

Reply:
not presently, but i'm trying to keep the linux/windows Dockerfiles as close as possible - they currently only differ in the `FROM` line for the final image: `scratch` or `windows/nanoserver`, and i am trying to figure out how to combine them in to a single multi-OS Dockerfile

Reply:
yeah that would nice if we can combine them into a single Dockerfile. Just curious, Is it possible to have a base dockerfile that can be included in both linux and windows dockerfile as they are using the same code

Reply:
maybe, i am currently investigating it